### PR TITLE
Rules: add new condition and new actions available in the UI

### PR DESCRIPTION
**Pull request by nmarniesse**
New condition:
- completeness filter  

New actions:
- ADD atribute values
- ADD groups
- new attribute types managed for ADD/SET attribute values: simple select, text and price collection

### CODE REVIEW

File: content/md/productivity/manage-your-rules.md [hunk 1]
```diff
@@ -102,8 +103,10 @@ Depending on the action you choose, the template changes. The table below lists
 | Clear categories  | Delete all the categories of your product/product model selection  |
 | Copy attribute value  | Copy the value from an attribute to another one  |
 | Remove categories  | Remove the categories of your product/product model selection  |
-| Set attribute value  | Set value to a multi select, number or boolean attribute. It will remove the former values |
+| Add attribute value  | Add values to a multi select, simple select, number, text, price collection or boolean attribute. It will keep the former values |
```
Comment (mmetayer) [inline content/md/productivity/manage-your-rules.md:106]: "add" action only works with multi-select, price collections, asset collections and reference entity multiple link

File: content/md/productivity/manage-your-rules.md [hunk 2]
```diff
@@ -102,8 +103,10 @@ Depending on the action you choose, the template changes. The table below lists
 | Clear categories  | Delete all the categories of your product/product model selection  |
 | Copy attribute value  | Copy the value from an attribute to another one  |
 | Remove categories  | Remove the categories of your product/product model selection  |
-| Set attribute value  | Set value to a multi select, number or boolean attribute. It will remove the former values |
+| Add attribute value  | Add values to a multi select or price collection. It will keep the former values |
```
Comment (meganelepalud-akeneo) [inline content/md/productivity/manage-your-rules.md:106]: price collection? "a multi select or a price attribute" maybe?

File: content/md/productivity/manage-your-rules.md [hunk 3]
```diff
@@ -102,8 +103,10 @@ Depending on the action you choose, the template changes. The table below lists
...
+| Set attribute value  | Set values to a multi select, simple select, number, text, price collection or boolean attribute. It will remove the former values |
```
Comment (meganelepalud-akeneo) [inline content/md/productivity/manage-your-rules.md:107]: same comment as the previous one. I don't see "price collection" as an attribute type.

Comment (nmarniesse) [reply content/md/productivity/manage-your-rules.md:107]: The "price" type is a collection of prices in multiple currencies (one price per actrivated currency). That's why we talk about an attribute of "price collection" type. I see "Price collection" in other pages in the helpcenter so I guess it's ok.

**Pull request merged**